### ISSUE TITLE
Confusion around accessibility query parameters and options

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/chipper#754]
This has come up a few times. For at least the sound and accessibility query parameters, the query parameter doesn't always overwrite the value of the sim option which creates some confusion. Also, it is confusing that we have two globals phet.chipper.accessibility and phet.chipper.queryparamters.accessibility. We have at least one incorrect usage of phet.chipper.queryparamters.accessibility.

We had some discussion in 5/9/19 meeting:
We could delete the phet.chipper.queryparamters.accessibility after it has been used to avoid all confusion.

At the very least we could have a comment in initialize-globals to never use phet.chipper.queryParameters.accessibility. We could write a lint rule that prevents us from using this one.

In the end, we decided we should just remove the global defined for accessibility in initialize-globals. Whenever required, we could reference phet.joist.sim.accessibility when we need to check globally.

**Comment by jessegreenberg** [comment]
The accessibility global in chipper has been removed. I think I did this right, @zepumph would you mind double checking?

I tested by making sure that ?a11y and ?accessibility params worked while the Sim.js accessibility flag still works the same. And I made sure places that were using the global are still working now that they are reaching into phet.joist.sim.accessibility.

**Comment by zepumph** [comment]
* Changes look nice. I like that everything is now based on a single field in Sim.
* `accessibility` is a bit of a strange name, and doesn't convey a boolean too well. Perhaps `sim.isAccessible` or `sim.a11yEnabled`. I don't feel too strongly about this, let me know what you think!

>We had some discussion in 5/9/19 meeting:
We could delete the phet.chipper.queryparamters.accessibility after it has been used to avoid all confusion.
At the very least we could have a comment in initialize-globals to never use phet.chipper.queryParameters.accessibility. We could write a lint rule that prevents us from using this one.

* Were we going to delete the a11y/accessibility query parameters to avoid confusion? Or at least document?

**Comment by jessegreenberg** [comment]
Thanks for the quick review.

> accessibility is a bit of a strange name, and doesn't convey a boolean too well

Sounds good, I like `isAccessible`.

> Were we going to delete the a11y/accessibility query parameters to avoid confusion? Or at least document?

Thanks, forgot about that. I am not sure I like deleting, but I will definitely add documentation.

**Comment by jessegreenberg** [comment]
OK, done above, anything else?

**Comment by zepumph** [comment]
The documentation looks excellet.

>if ( phet.chipper.queryParameters.fuzzBoard ) {
        assert && assert( this.isAccessible, 'fuzzBoard can only run with accessibility enabled.' );
        this.keyboardFuzzer.fuzzBoardEvents();
      }

This occurs before `this.isAccessible` is set to sim, likely fuzzBoard is broken right now.

**Comment by jessegreenberg** [comment]
That is called from within the `stepSimulationAction`, so it should be OK. I just tested fuzzBoard and it seems to be working.
